tests/PHPStan/Rules/Properties/PropertiesInInterfaceRuleTest.php

```diff
@@ -40,6 +40,10 @@ public function testPhp83AndPropertiesInInterface(): void
 				'Interfaces cannot include properties.',
 				11,
 			],
+			[
+				'Interfaces cannot include properties.',
```

Comment:
should this error also be changed to

```suggestion
				'Interfaces can include properties only on PHP 8.4 and later.',
```

?

Reply:
Yes please. When I originally added this error it was before PHP 8.4.